### ISSUE TITLE
rename the repo for idpbuilder to idplocal or localidpbuilder

### ISSUE THREAD

**Issue by nimakaviani** [cnoe-io/idpbuilder#52]
Given the conversation in this thread: https://cloud-native.slack.com/archives/C05TN9WFN5S/p1697488727377089?thread_ts=1697479794.107639&cid=C05TN9WFN5S

The name idpbuilder could be confusing as the current version builds a local idp and does not necessarily support building of a producation grade idp. hence the proposal to change the name of the repo.

vote on one of ones below:

**Comment by nimakaviani** [comment]
localidp

**Comment by nimakaviani** [comment]
localidpbuilder

**Comment by nimakaviani** [comment]
developmentIdp

**Comment by nimakaviani** [comment]
keep it as is

**Comment by cmoulliard** [comment]
What happens using as name `idplauncher` ?

**Comment by cmoulliard** [comment]
The word `build` or `builder` are also a bit confusing as we don't really build something but instead we setup a kind cluster locally where we deploy packages which, of course, are in relation with `backstage` and the concept about to setup an *I*nternal *D*eveloper *P*latform. 

So a better wording could be : 
```
./idplaunch create --name=local
./idplaunch delete --name=local 
```
or using such a few letters ;-)
```
./idp create --name=local --kubernetesVersion=v1.27.2`
./idp delete --name=local 
```

**Comment by greghaynes** [comment]
I fully support disambiguating this, but I am terrible at names (as evidenced by this current issue) so will defer to you all :).

**Comment by nimakaviani** [comment]
I think I am also leaning towards `localidp` based on the upvotes.

Also to @cmoulliard 's comment, the below also makes sense. Particularly because at the end of the day what we will have is a local preconfigured idp.

```
./localidp create --name=cnoe
./locealidp delete --name=cnoe 
```

**Comment by greghaynes** [comment]
SGTM!

**Comment by cmoulliard** [comment]
Why do you want to use the prefix `local` part of `./localidp` as the executable could be executed remotely too ?

**Comment by jessesanford** [comment]
Yeah I wonder if we are pigeonholing ourselves if we use the word "local" what about devidp? I believe the spirit of this tool is to be used in ideation process of integrating and creating idp capabilities. I don't think the goal is to always have everything locally, just that we want it to be very developer friendly.

**Comment by nimakaviani** [comment]
yeah you guys have a point. I was also thinking that while currently things are installed on a local cluster, later on you should be able to target a remote cluster as long as it has argo cd, argo workflows, and other stuff configured. maybe we dont rename it just yet and see how things go. saves us some refactoring time too.

**Comment by cmoulliard** [comment]
Why did you closed the ticket as no PR has been proposed ? @nabuskey

**Comment by nabuskey** [comment]
Oops my bad. I thought we agreed we are not changing the name. I was trying to clean up issues.

**Comment by cmoulliard** [comment]
Can we reopen the vote ? @greghaynes @jessesanford @nimakaviani @nabuskey 
I like a lot `idp` as CLI name as the name is short, logic, meaningful
